<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StocksControlsWidget</class>
 <widget class="QWidget" name="StocksControlsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>660</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>16777215</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="controlsVisiblePage">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>249</width>
            <height>614</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>4</number>
           </property>
           <property name="leftMargin">
            <number>4</number>
           </property>
           <property name="topMargin">
            <number>4</number>
           </property>
           <property name="rightMargin">
            <number>4</number>
           </property>
           <property name="bottomMargin">
            <number>4</number>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Price change from moment:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateTimeEdit" name="dateChangeTimeEdit">
             <property name="dateTime">
              <datetime>
               <hour>0</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2025</year>
               <month>1</month>
               <day>1</day>
              </datetime>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Filters</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <property name="spacing">
               <number>4</number>
              </property>
              <property name="leftMargin">
               <number>4</number>
              </property>
              <property name="topMargin">
               <number>4</number>
              </property>
              <property name="rightMargin">
               <number>4</number>
              </property>
              <property name="bottomMargin">
               <number>4</number>
              </property>
              <item>
               <widget class="QCheckBox" name="tickerCheckBox">
                <property name="text">
                 <string>Ticker</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="tickerLineEdit">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="qualInvestorCheckBox">
                <property name="text">
                 <string>Qual investor</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="qualInvestorComboBox">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="currentIndex">
                 <number>0</number>
                </property>
                <item>
                 <property name="text">
                  <string>Show all</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Only with status</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Only without status</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="priceCheckBox">
                <property name="text">
                 <string>Price</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QDoubleSpinBox" name="priceFromDoubleSpinBox">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="suffix">
                   <string notr="true"> p</string>
                  </property>
                  <property name="maximum">
                   <double>1000000.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string notr="true">-</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="priceToDoubleSpinBox">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="suffix">
                   <string notr="true"> p</string>
                  </property>
                  <property name="maximum">
                   <double>1000000.000000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QCheckBox" name="dayStartChangeCheckBox">
                <property name="text">
                 <string>Change from day start</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QDoubleSpinBox" name="dayStartChangeFromDoubleSpinBox">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="suffix">
                   <string notr="true">%</string>
                  </property>
                  <property name="minimum">
                   <double>-10000.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>10000.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string notr="true">-</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="dayStartChangeToDoubleSpinBox">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="suffix">
                   <string notr="true">%</string>
                  </property>
                  <property name="minimum">
                   <double>-10000.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>10000.000000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QCheckBox" name="dateChangeCheckBox">
                <property name="text">
                 <string>Change from some date</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QDoubleSpinBox" name="dateChangeFromDoubleSpinBox">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="suffix">
                   <string notr="true">%</string>
                  </property>
                  <property name="minimum">
                   <double>-10000.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>10000.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string notr="true">-</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="dateChangeToDoubleSpinBox">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="suffix">
                   <string notr="true">%</string>
                  </property>
                  <property name="minimum">
                   <double>-10000.000000000000000</double>
                  </property>
                  <property name="maximum">
                   <double>10000.000000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QCheckBox" name="turnoverCheckBox">
                <property name="toolTip">
                 <string>In thousands</string>
                </property>
                <property name="text">
                 <string>Turnover</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QSpinBox" name="turnoverFromSpinBox">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="suffix">
                   <string notr="true">K p</string>
                  </property>
                  <property name="maximum">
                   <number>1000000000</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string notr="true">-</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="turnoverToSpinBox">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="suffix">
                   <string notr="true">K p</string>
                  </property>
                  <property name="maximum">
                   <number>1000000000</number>
                  </property>
                  <property name="value">
                   <number>1000000000</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QCheckBox" name="paybackCheckBox">
                <property name="text">
                 <string>Payback</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QDoubleSpinBox" name="paybackFromDoubleSpinBox">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="suffix">
                   <string notr="true">%</string>
                  </property>
                  <property name="maximum">
                   <double>100.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string notr="true">-</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="paybackToDoubleSpinBox">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="suffix">
                   <string notr="true">%</string>
                  </property>
                  <property name="maximum">
                   <double>100.000000000000000</double>
                  </property>
                  <property name="value">
                   <double>100.000000000000000</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>645</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="controlsHiddenPage">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="VerticalLabel" name="filterActiveLabel">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string notr="true">Filter active</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QPushButton" name="hideButton">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../Resources.qrc">
         <normaloff>:/assets/images/left_arrows.png</normaloff>:/assets/images/left_arrows.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VerticalLabel</class>
   <extends>QLabel</extends>
   <header>src/verticallabel/verticallabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../Resources.qrc"/>
 </resources>
 <connections/>
</ui>
